arch/arc/Kconfig

```diff
@@ -38,6 +38,8 @@ config ARC
 	select HAVE_KERNEL_LZMA
 	select HAVE_KPROBES
 	select HAVE_KRETPROBES
+	select HAVE_FUNCTION_TRACER if (ISA_ARCV3)
```

Comment:
Do we really need brackets around `ISA_ARCV3`?

Reply:
No, I just thought it would be clearer with them. Took them out!